modules/manage/pages/manage-xdcr/create-xdcr-replication.adoc

```diff
@@ -137,6 +141,50 @@ image::manage-xdcr/xdcr-advanced-settings.png[,400,align=left]
 The values displayed in the fields are defaults, which can be modified interactively, and saved: this may help in achieving optimal replication-performance.
 For details on the significance of each field, see the xref:xdcr-reference:xdcr-reference-intro.adoc[XDCR Reference].
 
+[#identify-target-cluster-in-xdcr]
+=== Identify the Target Cluster in a Replication
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+This is recognized as target awareness and is used to display incoming replications on the target cluster.
+
+On the XDCR UI, the target cluster displays incoming replications, showing the source cluster and bucket information for each active replication.
+
+image::manage-xdcr/incoming-replications-on-target.png[,800,align=left]
+
+The information for the above incoming replications is retrieved using the REST API. See xref:rest-api:rest-xdcr-identify-target-cluster.adoc#complete-response[Cluster with replication specifications - a full example].
+
+To get the target awareness information using the REST API, see xref:rest-api:rest-xdcr-identify-target-cluster.adoc[Identify the Target Cluster in a Replication].
+
+==== Calculating Incoming XDCR Connections
+
+You can calculate the number of incoming XDCR connections for a replication with the following formula.
+
+----
+(target_nozzle_per_node) * 2 * (number of Data nodes on the source cluster) * (number of Data nodes on the target cluster)
+----
+
+Use this calculation to estimate the connection load on your target cluster.
+
+==== Target Awareness Heartbeat and Lag Time
+
+Target awareness relies on heartbeats sent from the source cluster to the target cluster.
+These heartbeats provide information about incoming replications.
+The incoming replication information is shown on the UI and returned by the `GET /xdcr/sourceClusters` REST API, as documented in xref:rest-api:rest-xdcr-identify-target-cluster.adoc[Identify the Target Cluster in a Replication].
+
+The interval for sending heartbeats is controlled by the target awareness heartbeat interval on the source cluster.
+By default, the heartbeat interval can range from one to five minutes, depending on replication activity.
+The heartbeat interval determines the refresh rate of the incoming replication information on the target cluster.
+
+Heartbeats are sent as soon as a remote reference is created, even if no replications exist yet.
```

Comment:
The source cluster heartbeats are sent as soon as the target remote cluster reference is created on the source, even if no replications exist yet.  So, in the incoming replications information, you may see the source cluster listed, but no replications may be listed if they have not been created yet.